convert/convert.go

```diff
@@ -13,9 +13,9 @@ import (
 	"strings"
 )
 
-func GetExistingTagsExpression(tokens hclwrite.Tokens) string {
+func GetExistingTagsExpression(tokens hclwrite.Tokens, tfVersion int) string {
 	if isHclMap(tokens) {
-		return buildMapExpression(tokens)
+		return buildMapExpression(tokens, tfVersion)
```

Comment:
BTW https://github.com/env0/terratag/issues/79
maybe would have been worth just doing that 🤷

Reply:
I think you're right. I was sure that `tomap` was needed here for the `merge`, but it isn't.
We can remove this buildMapExpression altogether probably, but that should only happen on a major version of terratag